users/views.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from .models import CustomUser
+from .serializers import ListUsersSerializer
+from rest_framework import generics
+from rest_framework.permissions import IsAdminUser
+
+class UsersListView(generics.ListAPIView):
```

Comment:
what's the reason behind this endpoint? Admin can still use the Django admin panel

Reply:
just add endpoint for : details of users .. you are right maybe  useless endpoint ... i will remove this one